cogs/info.py

```
@@ -0,0 +1,152 @@
+from nextcord.ext import commands
+from nextcord import Interaction, SlashOption
+
+
+class Info(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    
+
+    @nextcord.slash_command(description="Information about a user")
+    async def userinfo(self, i: Interaction, user: nextcord.Member = SlashOption(description='User to get info about', required=False)):
```

Ruff (0.11.9)

Do not perform function call `SlashOption` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)